test/suite/wttest.py

```diff
@@ -284,6 +317,33 @@ def __init__(self, *args, **kwargs):
         if not self._globalSetup:
             WiredTigerTestCase.globalSetup()
 
+    def tableExists(self, name):
+        # XXX This is a little ugly to have knowledge of specific hooks here.
```

Comment:
I suppose an alternate approach would be to allow hooks to over-ride or extend the implementation of `tableExists`. Then this logic would be part of the tiered hook and not the general test code.  

I have no idea if this is feasible, and it might wind up being just as ugly.